scripts/setup.sh

```diff
@@ -1,14 +1,54 @@
 #!/usr/bin/env bash
-
 set -ex
 
-apt-get update
-source /opt/ros/$ROS_DISTRO/setup.sh
+if [ -d "/opt/ros/melodic" ];
```

Comment:
this check looks a bit fragile to me.. it might be better to check if the existing ros installation is correct via `rosversion -d` or `dpkg -l | grep ros-melodic-*` or something similar.